metrics/metrics.go

```diff
@@ -93,6 +94,28 @@ func CollectProcessMetrics(refresh time.Duration) {
 		diskWriteBytes        = GetOrRegisterMeter("system/disk/writedata", DefaultRegistry)
 		diskWriteBytesCounter = GetOrRegisterCounter("system/disk/writebytes", DefaultRegistry)
 	)
+
+	option := profiler.WithProfileTypes(
```

Comment:
Can we not enable datadog by default? Many node runners might not use it.

We can add command `--metrics.datadog` and `--metrics.datadog.expensive`

Reply:
That was my first intention https://github.com/maticnetwork/bor/pull/338/commits/9a62679609622fca9c4b0236e96cb85d0efc4e26 to put it under the `pprof` config, but it didn't work, going to look to it carefully now, and add the config param.